src/chunkserver/datastore/chunkserver_chunkfile.cpp

```diff
@@ -74,56 +75,130 @@ ChunkFileMetaPage& ChunkFileMetaPage::operator =(
     return *this;
 }
 
-void ChunkFileMetaPage::encode(char* buf) {
+uint64_t ChunkFileMetaPage::htonll(uint64_t val) {
```

Comment:
use the functions defined in the endian.h header file, or completely implement the endian conversion functions by youself.